docs/src/examples/components/Menu/Slots/MenuExample.shorthand.tsx

```diff
@@ -7,6 +7,6 @@ const items = [
   { key: 'events', content: 'Upcoming Events' },
 ]
 
-const MenuExample = () => <Menu defaultActiveIndex={0} items={items} />
+const MenuExample = () => <Menu defaultActiveIndex={0} items={items} primary />
```

Comment:
Why are we adding primary on all of the examples? I know that in Teams theme it is more used, but with this example we are laying of what we are showing (we say default Menu, when indeed we are showing the primary variations)

Reply:
I know, good that you opened this discussion; I want to see screener regression for teams theme otherwise they are not caught at all, what's the alternative here?

Reply:
See my comment: https://github.com/stardust-ui/react/pull/983#discussion_r261301783

Reply:
### So it looks like we came again to the discussion about screener visual tests and using the examples as input.

For this PR I need more things to be tested than what we have examples for; for instance I need to test all menus with `primary` prop being supplied. However, we might not want to change all examples to have `primary` prop (which is what I did in this PR) just so that screener will create screenshots for the primary menus.

Best thing would be to move all screener related stuff from  `docs/src/examples/` somewhere under `tests/screener/` or `tests/screenshots`.

For now I'm thinking of following @mnajdova 's suggestions and add example files that I'm not exporting in `index.tsx` so that they are not being shown on the docs site.

I'm also considering marking them with `.test.tsx` extension and write the steps in `test.stetps.ts`

e.g. this file will be reverted and we'll have:
- `docs/src/examples/components/Menu/Slots/MenuExample.shorthand.tsx` -> this file reverted
`docs/src/examples/components/Menu/Slots/MenuExamplePrimary.test.tsx` -> menu with `primary` prop used for screenshot tests but not exported in `index.tsx`
`docs/src/examples/components/Menu/Slots/MenuExamplePrimary.steps.ts` -> steps for `MenuExamplePrimary.test.tsx`

It'd be great if we could add these steps directly in `MenuExamplePrimary.test.tsx` so that we don't create more files?

@mnajdova 
@silviuavram 
@layershifter 
@kuzhelov 
@miroslavstastny

Reply:
extracted in #1000